src/main/java/org/reactome/release/qa/check/CoV2InfectionPathwayEventCheck.java

```diff
@@ -0,0 +1,191 @@
+       for (GKInstance summation : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.summation)) {
+            GKInstance createdInst = (GKInstance) summation.getAttributeValue(ReactomeJavaConstants.created);
+            List<GKInstance> modifieds = summation.getAttributeValuesList(ReactomeJavaConstants.modified);
+            if (modifieds.size() > 0) {
+                // At time of writing (August 2020), nothing is reported from this clause, but it is a good check.
+                GKInstance mostRecentModifiedInst = modifieds.get(modifieds.size() - 1);
+                // Only check Year/Month/Day since the changes took place more than a day before Curators got to look at them in gk_central.
+                LocalDate createdDateTime = getDateTimeFromInstance(createdInst);
+                LocalDate modifiedDateTime = getDateTimeFromInstance(mostRecentModifiedInst);
+                if (createdDateTime.isAfter(modifiedDateTime)) {
+                    return false;
+                }
+            // Should be at least 1 modified instance in CoV-1-to-CoV-2 instances.
+            } else {
+                return false;
+            }
+       }
+        return true;
+    }
+
+    /**
+     * Parses out the Date from dateTime attribute of the incoming InstanceEdit instance type.
+     * This value is formatted as 'yyyy-MM-dd hh:mm:ss', and the 'date' portion (1st half) is all that is needed for this test.
+     * @param instanceEditInst - GKInstance, from either the 'created' or 'modified' attribute of an instance.
+     * @return - LocalDate, in the format 'yyyy-MM-dd'.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private LocalDate getDateTimeFromInstance(GKInstance instanceEditInst) throws Exception {
+        return LocalDate.parse(instanceEditInst.getAttributeValue(ReactomeJavaConstants.dateTime).toString().split(" ")[0]);
+    }
+
+    /**
+     * Checks if Event has a 2020 literatureReference. This is because CoV-2 evidence was very recent when this was being evaluated.
+     * @param cov2Event - GKInstance, Event contained within 'SARS-CoV-2 Infection' Pathway.
+     * @return - boolean, true if at least 1 2020 literatureReference, false if not.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private boolean hasRecentLiteratureReference(GKInstance cov2Event) throws Exception {
+
+        boolean has2020LiteratureReference = false;
+        // Iterature through all literatureReference instances, checking the 'year' attribute.
+        for (GKInstance literatureReference : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.literatureReference)) {
+            if (literatureReference.getSchemClass().isa(ReactomeJavaConstants.LiteratureReference)) {
+                if (literatureReference.getAttributeValue(ReactomeJavaConstants.year).toString().equals("2020")) {
```

Comment:
Could these two be combined into a single if-statement?
```java
if (literatureReference.getSchemClass().isa(ReactomeJavaConstants.LiteratureReference)
    &&  literatureReference.getSchemClass().isa(ReactomeJavaConstants.LiteratureReference))
{
...
```

Reply:
But don't you find that find that redundancy helps you be sure be sure?

Reply:
Lol

Reply:
Lol

Reply:
OK fine... here's what I meant:
```java
if (literatureReference.getSchemClass().isa(ReactomeJavaConstants.LiteratureReference)
    && literatureReference.getAttributeValue(ReactomeJavaConstants.year).toString().equals("2020"))
{
```

Reply:
Wow I need to practice my reading comprehension - I thought I had accidentally done the same 'if' statement twice in a row. My bad! I have implemented the && approach (and removed break statements in this method (referring to https://github.com/reactome/release-qa/pull/26#discussion_r534280332)